src/main/java/com/epam/processor/DataProcessor.java

```diff
@@ -27,6 +26,12 @@ public DataProcessor(List<RoadAccident> roadAccidentList){
      * @return
      */
     public RoadAccident getAccidentByIndex7(String index){
+        Iterator<RoadAccident> iter = roadAccidentList.iterator();
+        while(iter.hasNext()) {
```

Comment:
I would recomend to use for(accident: list) kind of syntax - no need to create iterator by yourself.